main.py

```diff
@@ -43,8 +43,11 @@ def makeMosaic(im, keyword, tile_size, group_size, verbose=False):
 		y1 = y2 + 1
 		y2 += tile_size + 1
 
-	mosaic.show()
+	mosiac_file_name = file_name.split('.')[0] + '_mosaic.png'
+
+	mosaic.save(mosiac_file_name, 'PNG')
```

Comment:
why png?

Reply:
`mosaic.show()` opened as a PNG so I thought that's a good default image type.